example_tb/core/Makefile

```diff
@@ -88,9 +90,9 @@ vlib: .lib-rtl
 	touch .lib-rtl
 
 # rebuild if we change some sourcefile
-.build-rtl: .lib-rtl $(RTLSRC_PKG) $(RTLSRC) $(RTLSRC_BHV_PKG) $(RTLSRC_BHV) $(RTLSRC_TB_PKG) $(RTLSRC_TB)
-	$(VLOG) -work $(VWORK) +incdir+$(RTLSRC_INCDIR) +incdir+$(RTLSRC_BHV_INCDIR) $(VLOG_FLAGS) \
-	$(RTLSRC_PKG) $(RTLSRC) $(RTLSRC_BHV_PKG) $(RTLSRC_BHV) $(RTLSRC_TB_PKG) $(RTLSRC_TB)
+.build-rtl: .lib-rtl $(CV_CORE_MANIFEST) $(RTLSRC_BHV) $(RTLSRC_TB_PKG) $(RTLSRC_TB)
```

Comment:
RTLSRC_BHV should not be needed here as it is in the manifest file. Only manifest allowed here plus TB files

Reply:
correct